cheatsheets/Error_Handling_Cheat_Sheet.md

```diff
@@ -1,16 +1,16 @@
 # Introduction
 
-Error handling is a part of the overall security of an application. Except in movies, an attack always begin by a **Reconnaissance** phase in which the attacker will try to gather as many technical information (often *name* and *version* properties) as possible about the target like for example the application server, the technologies, the frameworks, the libraries...
+Error handling is a part of the overall security of an application. Except in movies, an attack always begin by a **Reconnaissance** phase in which the attacker will try to gather as much technical information (often *name* and *version* properties) as possible about the target like, for example, the application server, the technologies, the frameworks, the libraries...
 
-So, if errors are not well handled then it can highly facilitate the work for an attacker into this initial phase that is very important for the rest of the whole attack.
+Unhandled errors facilitate the work for an attacker in this initial phase, which is very important for the rest the attack.
```

Comment:
`facilitate the work for an attacker` is another slightly odd bit of phrasing. Maybe `...can assist an attacker in this initial phase...`?